.github/ISSUE_TEMPLATE/blog.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Blog Post
+about: Blog website article template
+title: '[BLOG] <blog-title>'
+labels: 'blog'
+assignees: ''
+
+---
+
+<!-- Thank you for contributing to the TAG!
+    Please use this issue template only if you are planning to write a blog post for the TAG Environmental Sustainability.
+    The blog can be found here: https://tag-env-sustainability.cncf.io/blog/
+-->
+
+- **Authors**: *TBD*
+- **TAG ENV Representative**: *TBD*
+- **Reviewers**: *TBD*
+
+<!--
+Authors: anyone can propose to write a blog post which is published to the TAG ENV blog
+TAG ENV Representative: is either a TAG ENV chair or TL
+Reviewers: anyone can be a reviewer of the blog post
+-->
+
+## Outline
+
+<!--
+Please summarize what this blog post is about
+-->
+
+*TBD*
```

Comment:
This looks really good.

It would be useful, when evaluating a post, on who the "Expected audience" is.  What assumptions are we making for the audience of this blog post?

Also, goals.
"what do we expect the audience to get out of this."

And non-goals:
"what do we want to be careful of being sensitive about and not implying opinion on"

Reply:
Thanks, added